expertise/models/multifacet_recommender/specter.py

```diff
@@ -58,10 +61,13 @@ def __init__(self,
                  output_file: Optional[str],
                  batch_size: int,
                  print_to_console: bool,
-                 has_dataset_reader: bool) -> None:
+                 has_dataset_reader: bool,
+                 store_redis: bool = False) -> None:
         super(_PredictManagerCustom, self).__init__(predictor, input_file, output_file, batch_size, print_to_console,
                                                     has_dataset_reader)
         self.total_size = int(sum([1 for _ in open(self._input_file)]) / self._batch_size)
+        self._store_redis = store_redis
+        self._redis_con = redisai.Client(host='localhost', port=6379, db=5)
```

Comment:
is it possible to pass the Redis client as a parameter in the constructor?

Reply:
I believe so, but I'd like to keep this client here itself, as this is `RedisAI` client which will only be used by specter.

Reply:
Yes, but you can create the client outside of this object and then this is easier to test and you don't need to read from a configuration file here to know the redis connection args.

Reply:
yes, that can be done.